core/src/main/java/io/aiven/klaw/controller/UsersTeamsController.java

```diff
@@ -58,6 +60,7 @@ public ResponseEntity<List<String>> getAllTeamsSUOnly() {
     return new ResponseEntity<>(usersTeamsControllerService.getAllTeamsSUOnly(), HttpStatus.OK);
   }
 
+  @PermissionAllowed(permissionAllowed = {PermissionType.ADD_EDIT_DELETE_TEAMS})
```

Comment:
do we also want to add or FULL_ACCESS_USERS_TEAMS_ROLES

Reply:
Yes good catch thank you!